griptape/loaders/base_file_loader.py

```diff
@@ -27,3 +27,9 @@ def fetch(self, source: str | PathLike) -> bytes:
             return data.encode(self.encoding)
         else:
             return data
+
+    def save(self, destination: str | PathLike, artifact: A) -> None:
+        """Saves the Artifact to a destination."""
+        data = artifact.value if isinstance(artifact.value, bytes) else artifact.value.encode(self.encoding)
```

Comment:
This is a pretty general serialization operations, seems it would be good in a `to_bytes` method.

Reply:
That reminds me I already added one! Updated.